cli.js

```diff
@@ -80,7 +80,7 @@ var cli = meow([
 	'  ava --init foo.js',
 	'',
 	'Default patterns when no arguments:',
-	'test.js test-*.js test/**/*.js'
+	'test.js test-*.js test/**/*.js **/__tests__ **/*.test.js'
```

Comment:
Documentation wise it should be `**/__tests__/**/*.js`, just like the `test` directory in `ava-files` is explained as `test/**/*.js`.